mlir-tensorrt/executor/include/mlir-executor-c/Runtime/Runtime.h

```diff
@@ -375,15 +386,22 @@ static inline bool mtrtRuntimeSessionIsNull(MTRT_RuntimeSession session) {
 }
 
 /// Using `session`, execute the pubic function with the specified name.
-/// The `inArgs` and `outArgs` are arrays for input arguments and destination
-/// arguments, respectively. Input arguments may be MemRefs or scalars, but
-/// destination arguments must be MemRefs.
-/// A stream may optionally be specified, otherwise pass the result of
-/// `mtrtStreamGetNull()`.
+/// The `inArgs`, `outArgs`, and `results` are arrays for input arguments and
+/// optional destination arguments and results, respectively. Input arguments
+/// may be MemRefs or scalars, but destination arguments and results must be
```

Comment:
It should not be required that `results` are MemRefs. This is not required by the compiler. For example, the functions we often use for testing with `mlir-tensorrt-runner` usually return an integer code a'la normal C/C++ `main` functions.